alacritty/src/event.rs

```diff
@@ -274,193 +317,195 @@ impl ApplicationHandler<Event> for Processor {
             &self.proxy,
             &mut self.clipboard,
             &mut self.scheduler,
-            WinitEvent::WindowEvent { window_id, event },
+            WinitEvent::WindowEvent(event),
         );
 
         if is_redraw {
             window_context.draw(&mut self.scheduler);
         }
     }
 
-    fn user_event(&mut self, event_loop: &ActiveEventLoop, event: Event) {
-        if self.config.debug.print_events {
-            info!(target: LOG_TARGET_WINIT, "{event:?}");
-        }
+    fn proxy_wake_up(&mut self, event_loop: &dyn ActiveEventLoop) {
+        while let Ok(event) = self.user_events_rx.try_recv() {
```

Comment:
I'd assume you're intentionally ignoring the error here because the channel should never be able to shut down on the sending side?

Reply:
Yeah, and how it was before.